CONTRIBUTING.md

```diff
@@ -132,21 +132,21 @@ For issues related to continuous testing workflows or multi-component "epics" st
 #### <a name='Tools'></a>Tools
 
 - [cFS-GroundSystem](https://github.com/nasa/cFS-GroundSystem/issues)
-- [elf2cfetbl](https://github.com/nasa/elf2cfetbl/tree/6762b1c3b455665dae57e35f14a50fe327830391)
-- [tblCRCTool](https://github.com/nasa/tblCRCTool/tree/b02864ba56b12e00ab152225e3e8f9d6c039d48c)
+- [elf2cfetbl](https://github.com/nasa/elf2cfetbl/issues)
+- [tblCRCTool](https://github.com/nasa/tblCRCTool/issues)
 
 #### <a name='OthercFSApps'></a>Other cFS Apps
 
-- [BP](https://github.com/nasa/bp/issues)
-- [CF](https://github.com/nasa/cf/issues)
-- [CS](https://github.com/nasa/cs/issues)
-- [DS](https://github.com/nasa/ds/issues)
-- [FM](https://github.com/nasa/fm/issues)
-- [HK](https://github.com/nasa/HK//issues)
-- [HS](https://github.com/nasa/HS//issues)
-- [LC](https://github.com/nasa/LC//issues)
-- [MD](https://github.com/nasa/MD//issues)
-- [MM](https://github.com/nasa/MM//issues)
+- [BP](https://github.com/nasa/BP/issues)
+- [CF](https://github.com/nasa/CF/issues)
+- [CS](https://github.com/nasa/CS/issues)
+- [DS](https://github.com/nasa/DS/issues)
+- [FM](https://github.com/nasa/FM/issues)
+- [HK](https://github.com/nasa/HK/issues)
+- [HS](https://github.com/nasa/HS/issues)
+- [LC](https://github.com/nasa/LC/issues)
+- [MD](https://github.com/nasa/MD/issues)
+- [MM](https://github.com/nasa/MM/issues)
 - [SC](https://github.com/nasa/SC/issues)
 
```

Comment:
Consider adding SCH, git commit --amend, and git push --force

- [SCH](https://github.com/nasa/SCH//issues)

Reply:
Added.